packages/loader/src/resource-deserialize/resources/schema/BasicSchema.ts

```diff
@@ -28,7 +28,15 @@ export interface IHierarchyFile {
   entities: Array<IEntity>;
 }
 
-export type IMethodParams = Array<IBasicType>;
+export type ICanParseResultObject = {
```

Comment:
It's a awful name :joy:

Reply:
How about:  IMethodParamsAndResInfo

type IMethodInfo = Array<IBasicType> | IMethodParamsAndResInfo

Reply:
Use markdown and pay attention to format of typescript code.

Reply:
ok, opted after discuss